tests/kernel/run_ltp.pm

```diff
@@ -238,9 +239,34 @@ sub record_ltp_result {
     }
 
     $self->commit_result($details, $fh);
+    $self->write_extra_test_result($export_details);
     return (0, $export_details);
 }
 
+sub write_extra_test_result {
+    my ($self, $details) = @_;
+    my $dir      = bmwqemu::result_dir();
+    my $filename = $details->{test_fqn} =~ s/:/_/gr;
+    my $result   = 'failed';
+    $result = 'passed'  if ($details->{test}->{result} eq 'PASS');
+    $result = 'skipped' if ($details->{test}->{result} eq 'CONF');
+
+    my $result_file = {
+        dents   => 0,
+        details => [{
+                _source => 'parser',
+                result  => $result =~ s/ed$//r,
```

Comment:
I just noticed that this doesn't work. Then the result on the "External results" page isn't displayed properly.

Reply:
@Martchus can you help me here. I would like to have the detail step also rendered in the corresponding color. But I'm not sure, what I need to set as result.